frontend/src/views/MovieDetailsScreen.vue

```diff
@@ -42,7 +42,7 @@ export default {
 
     async fetchMovie (id) {
       try {
-        const { data: movie } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movies-${id}`, {
+        const { data: movie } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movies/${id}`, {
```

Comment:
You should also add slashes at the end of the url, this is why errors happen on the detail pages